src/test/java/io/kokuwa/maven/helm/UploadMojoTest.java

```diff
@@ -286,6 +288,58 @@ void inputRepositoryTypeRequired(UploadMojo mojo) throws Exception {
 		assertThrows(IllegalArgumentException.class, mojo::execute, "Missing repo type must fail.");
 	}
 
+	@DisplayName("flag: with flag verify")
+	@Test
+	void verify(UploadMojo mojo) throws IOException {
+		mojo.setUploadVerification(true);
+		mojo.setChartVersion("0.1.0");
+		mojo.setUploadRepoStable(new HelmRepository()
+				.setType(RepoType.NEXUS)
+				.setName("my-nexus")
+				.setUrl("http://127.0.0.1:" + mock.getPort() + "/nexus"));
+		Path packaged = copyPackagedHelmChartToOutputdirectory(mojo);
+		assertUploadVerifySuccess(mojo, RequestMethod.PUT, "/nexus/" + packaged.getFileName());
+	}
+
+	@DisplayName("flag: with flags verify and timeout")
+	@Test
+	void verifyAndTimeout(UploadMojo mojo) {
+		mojo.setUploadVerification(true);
+		mojo.setUploadVerificationTimeout(10);
+		mojo.setChartVersion("0.1.0");
+		mojo.setUploadRepoStable(new HelmRepository()
+				.setType(RepoType.NEXUS)
+				.setName("my-nexus")
+				.setUrl("http://127.0.0.1:" + mock.getPort() + "/nexus"));
+		Path packaged = copyPackagedHelmChartToOutputdirectory(mojo);
+		assertUploadVerifySuccess(mojo, RequestMethod.PUT, "/nexus/" + packaged.getFileName());
+	}
+
+	@DisplayName("flag: with flags verify and timeout times out")
+	@Test
+	void verifyAndTimeoutFail(UploadMojo mojo) {
+		mojo.setUploadVerification(true);
+		mojo.setUploadVerificationTimeout(10);
+		mojo.setChartVersion("0.1.0");
+		mojo.setUploadRepoStable(new HelmRepository()
+				.setType(RepoType.NEXUS)
+				.setName("my-nexus")
+				.setUrl("http://127.0.0.1:" + mock.getPort() + "/nexus"));
+		mojo.setChartDirectory(new File("src/test/resources/simple-fail/"));
```

Comment:
simple-fail/ holds a chart that is not stubbed to be held in the mock repository, hence this will cause the verification to fail in this test case